content/en/news/releases/1.6.x/announcing-1.6.3/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Announcing Istio 1.6.3
+linktitle: 1.6.3
+subtitle: Patch Release
+description: Istio 1.6.3 patch release.
+publishdate: 2020-06-18
+release: 1.6.3
+aliases:
+    - /news/announcing-1.6.3
+---
+
+This release contains bug fixes to improve robustness. This release note describes
+what’s different between Istio 1.6.2 and Istio 1.6.3.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** an issue preventing the operator from recreating watched resources if they are deleted ([Issue 23238](https://github.com/istio/istio/issues/23238)).
+- **Fixed** an issue where Istio crashed with the message: `proto.Message is *client.QuotaSpecBinding, not *client`([Issue 24624](https://github.com/istio/istio/issues/24264)).
+- **Fixed** an issue preventing operator reconciliation due to improper labels on watched resources ([Issue 23603](https://github.com/istio/istio/issues/23603)).
+- **Added** support for the `k8s.v1.cni.cncf.io/networks` annotation ([Issue 24426](https://github.com/istio/istio/pull/24426)).
+- **Updated** the `SidecarInjectionSpec` CRD to read the `imagePullSecret` from `.Values.global` ([Issue 24437](https://github.com/istio/istio/issues/24437)).
```

Comment:
Maybe update to point at https://github.com/istio/istio/pull/24365.

Reply:
Back in 1.3, we had the decision to only point at issues, not pull requests. Do we want to point at pull requests for 1.6? @jacob-delgado any thoughts?

Reply:
Fixed. Thanks.